### PR TITLE
Generate layers string with `create_layers` function

### LINKED ISSUE #43: Need a way to construct layer strings
Layer strings tell API endpoints which telescope and instrument to use, for example 13 => SDO AIA 304
We'll need a mapping of these, maybe an enum for users to select data sources.

Also Layer strings can have multiple sources so that the final image is a composition of the given layers.
i.e. [8,1,50],[13,1,50] will return a composite of AIA 94 and AIA 304 each with 50% opacity layered on top of each other.

### PR DESCRIPTION
**Pull request by akash5100**
* closes #43

### CODE REVIEW

File: hvpy/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+def test_create_layers():
+    assert create_layers([(9, 100), (11, 50)]) == "[9,1,100],[11,1,50]"
+
+    with pytest.raises(ValueError, match="opacity: -1 must be between 0 and 100"):
+        create_layers([(9, 100), (14, -1)])
+
+    with pytest.raises(ValueError, match="123 is not a valid DataSources"):
+        create_layers([(9, 100), (123, 100)])
```
Comment (dgarciabriseno) [inline hvpy/tests/test_utils.py:13]: Can you use 999 instead of 123, I feel like we might actually reach 123 soon

File: docs/api.rst [hunk 2]
```diff
@@ -15,3 +15,7 @@ API Reference
 
 .. automodapi:: hvpy.datasources
    :no-inheritance-diagram:
+
+.. automodapi:: hvpy.utils
+   :no-inheritance-diagram:
+   :members: create_layers
```
Comment (nabobalis) [inline docs/api.rst:21]:
```suggestion
```

File: hvpy/tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from hvpy import create_layers
+
+
+def test_create_layers():
+    assert create_layers([(9, 100), (11, 50)]) == "[9,1,100],[11,1,50]"
+
+    with pytest.raises(ValueError, match="opacity: -1 must be between 0 and 100"):
+        create_layers([(9, 100), (14, -1)])
```
Comment (dgarciabriseno) [inline hvpy/tests/test_utils.py:10]: Can you also test 101 as an invalid opacity, to test both sides.

File: hvpy/utils.py [hunk 4]
```diff
@@ -29,3 +36,43 @@ def convert_date_to_unix(v: list) -> str:
...
+
+
+def create_layers(layer: list) -> str:
+    """
+    Creates a string of layers separated by commas.
+
+    Parameters
+    ----------
+    layer
+        A list of tuples of the form (source_id, opacity).
```
Comment (nabobalis) [inline hvpy/utils.py:70]:
```suggestion
        A list of tuples of the form (``source_id``, ``opacity``).
```

File: hvpy/utils.py [hunk 5]
```diff
@@ -29,3 +36,43 @@ def convert_date_to_unix(v: list) -> str:
...
+    else:
+        raise ValueError(f"{val} is not a valid DataSources")
+
+
+def _create_layer_string(source_id: DataSources, opacity: int) -> str:
+    """
+    Generates a string of the form "[source_id,1,opacity]" for a layer.
+    """
+    if not 0 <= opacity <= 100:
+        raise ValueError(f"opacity: {opacity} must be between 0 and 100")
```
Comment (nabobalis) [inline hvpy/utils.py:58]:
```suggestion
        raise ValueError(f"opacity ({opacity}) must be between 0 and 100")
```

**Review by nabobalis — APPROVED** [review]

**Pull request merged**